js/rpg_core/Bitmap.js

```diff
@@ -858,8 +932,9 @@ Bitmap.prototype._requestImage = function(url){
         Decrypter.decryptImg(url, this);
     } else {
         this._image.src = url;
-        this._image.onload = Bitmap.prototype._onLoad.bind(this);
-        this._image.onerror = Bitmap.prototype._onError.bind(this);
+
+        this._image.addEventListener('load', this._loadListener = Bitmap.prototype._onLoad.bind(this));
+        this._image.addEventListener('error', this._errorListener = Bitmap.prototype._onError.bind(this));
```

Comment:
`Decrypter.decryptImg` used `onload` and `onerror` yet.
It would be nice to unify how to set listener between `Bitmap#_requestImage` and `Decrypter.decryptImg`.
(Again, I don't know why you replaced `onXxx` to `addEventListener`. ❓ )